EIPS/eip-nnnn.md

```diff
@@ -0,0 +1,83 @@
+---
+eip: nnnn
```

Comment:
```suggestion
eip: 7698
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.